### PR TITLE
WIP: Z39

### PR DESCRIPTION
**Pull request by PXke**
Proposition for legacy integration of Z39

### CODE REVIEW

**Comment by PXke** [comment]
Hello @tiborsimko all feedbacks are welcome. It is our first time we develop a new module for legacy.

File: modules/bibz39/Makefile [hunk 1]
```diff
@@ -0,0 +1,646 @@
+# Makefile.in generated by automake 1.14.1 from Makefile.am.
```
Comment (kaplun) [inline modules/bibz39/Makefile]: @PXke you should not commit the `Makefile(.in)` files, but only `Makefile.am`. The former are generated from the latter by `automake` stack.

File: requirements.txt [hunk 2]
```diff
@@ -29,3 +29,5 @@ redis==2.9.0
 nydus==0.10.6
 Cerberus==0.5
 matplotlib
+git+https://github.com/asl2/PyZ3950.git
```
Comment (kaplun) [inline requirements.txt]: PyZ3950 is on pypi and latest version corresponds to the github master. So you can simply put `PyZ3950==2.04`

File: modules/bibz39/Makefile (hunk 1 above)
Comment (PXke) [inline modules/bibz39/Makefile]: Thank you !

File: requirements.txt (hunk 2 above)
Comment (PXke) [inline requirements.txt]: Thank you !

**Comment by kaplun** [comment]
Hi! Is this still a WIP?

**Comment by PXke** [comment]
Sorry @kaplun but yes, QA failed on our side so there is still work done on this module. It should be finished today or tomorrow though as we are reaching the end of the process.

**Comment by kaplun** [comment]
No problem! Thanks for feedback :smile_cat:

**Pull request closed without merging**